chrome/content/test/globals.js

```diff
@@ -6,33 +6,93 @@
 const PrivlyXPCOM = Components.classes["@privly.test/jetpack;1"].
                       getService(Components.interfaces.nsISupports).
                       wrappedJSObject;
+
 // Global object
 // Provides access to Jetpack API's and Add-on code CommonJS modules for unit testing.
 this.g = {
 
-  // Local Storage Shim
-  ls: PrivlyXPCOM.getLocalStorageShim().ls,
+  /**
+   * First run
+   */
+  firstRun: PrivlyXPCOM.getFirstRun().firstRun,
 
-  // Jetpack Preferences service object
-  preferences: PrivlyXPCOM.getPreferences(),
-  
-  // Extension version
-  version: PrivlyXPCOM.getSelfObject().version,
- 
-  // Extension load reason - install, upgrade, downgrade, update.
+  /**
+   * Extension load reason - install, upgrade, downgrade, update.
+   */
   loadReason: PrivlyXPCOM.getSelfObject().loadReason,
 
-  // Jetpack tabs object
+  /**
+   * Jetpack tabs object
+   */
   tabs: PrivlyXPCOM.getTabs(),
 
-  // Jetpack windows object
+  /**
+   * Jetpack windows object
+   */
   windows: PrivlyXPCOM.getWindows(),
 
-  // Privly UI Popup Button
+  /**
+   * Privly UI Popup Button
+   */
   popupButton: PrivlyXPCOM.getPrivlyUI().popupButton,
  
-  // Privly Posting Process
+  /**
+   * Privly Posting Process
+   */
   postingProcess: PrivlyXPCOM.getPostingProcess().postingProcess,
```

Comment:
This is where we will have issues with the Xul deprecation because all the globals are defined via xpcom?